### PR TITLE
[[ Bug 19535 ]] Fixed bug causing incorrect drawing of rectangle when…

### PR DESCRIPTION
**Pull request by abrisan**
… cmd-a

### CODE REVIEW

File: engine/src/paragraf.cpp [hunk 1]
```diff
@@ -1265,7 +1265,7 @@ void MCParagraph::fillselect(MCDC *dc, MCLine *lptr, int2 x, int2 y, uint2 heigh
                     ei = bi + bl;
                 
                 // Get the X coordinates for the selection
-                int2 bix, bex;
+                findex_t bix, bex;
```
Comment (runrevmark) [inline engine/src/paragraf.cpp]: findex_t isn't the right type to use here - as bix and bex are co-ords - i.e. coord_t

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 9e1ea42

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 9e1ea42c11a063cf4112bfe0072019f21d77dadf

**Comment by livecode-vulcan** [comment]
😎 test success 9e1ea42c11a063cf4112bfe0072019f21d77dadf

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/338)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/344)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/324)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/267)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/267)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/333)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/338)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/336)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/342)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/336)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/342)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/361)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/332)

**Pull request merged**